lib/serializer.js

```diff
@@ -94,6 +97,10 @@ module.exports = class Serializer {
       str = str.toString()
     }
 
+    if (!STR_ESCAPE.test(str)) {
```

Comment:
I was wondering, if following has a perf impact?
```suggestion
    if (STR_ESCAPE.test(str) === false) {
```

Could you please bench it?

Reply:
It doesn't.

Reply:
Thx